src/tests/activities.test.js

```diff
@@ -1,3 +1,130 @@
+  isActive: true,
+};
+
+const ActivityBad = {
+  name: '',
+  description: '',
+  isActive: true,
+};
+
+const updatedActivity = {
+  name: 'basquetball',
+  description: 'This is the most practiced sport nowadays by teams of men and women, both professionally and amateurishly.',
+  isActive: false,
+};
+
+beforeAll(async () => {
+  await activity.collection.insertMany(activitySeed);
+});
+
+describe('POST /api/activities', () => {
+  test('should create a activity', async () => {
+    const res = await request(app).post('/api/activities').send(activities);
+    expect(res.status).toBe(201);
+    expect(res.body).toBeDefined();
+    expect(res.error).toBeFalsy();
+  });
+
+  test('should return a error', async () => {
+    const res = await request(app).post('/api/activities').send(ActivityBad);
+    expect(res.status).toBe(400);
+    expect(res.body.data).toBeUndefined();
+    expect(res.error).toBeTruthy();
+  });
+});
+
+describe('PUT /api/activities/:id', () => {
+  test('should update a activity', async () => {
+    const res = await request(app).put('/api/activities/6466b3e7a21fd4069bcaf7c0').send(updatedActivity);
+    expect(res.status).toBe(200);
+    expect(res.body).toBeDefined();
+    expect(res.error).toBeFalsy();
+  });
+
+  test('should return a error', async () => {
+    const res = await request(app).put('/api/activities/6466b3e7a21fd4069bcaf7c0').send(ActivityBad);
+    expect(res.status).toBe(400);
+    expect(res.body.data).toBeUndefined();
+    expect(res.error).toBeTruthy();
+  });
+
+  test('should return a error, id not valid', async () => {
+    const res = await request(app).put('/api/activities/6462261').send(updatedActivity);
+    expect(res.status).toBe(400);
+    expect(res.error).toBeTruthy();
+  });
+});
+
+describe('DELETE /api/activities/:id', () => {
+  test('should delete a activity', async () => {
+    const res = await request(app).delete('/api/activities/6467cd965eada13a19071ab9');
+    expect(res.status).toBe(200);
+    expect(res.body).toBeDefined();
+    expect(res.error).toBeFalsy();
+  });
+
+  test('should return error, id not valid', async () => {
+    const res = await request(app).delete('/api/activities/6467cd9');
+    expect(res.status).toBe(400);
+    expect(res.error).toBeTruthy();
+  });
+
+  test('should return error, id not found', async () => {
+    const res = await request(app).delete('/api/activities/6467cd965eada13a19071ab5');
+    expect(res.status).toBe(404);
+    expect(res.body.data).toBeUndefined();
+    expect(res.error).toBeTruthy();
+  });
+});
+
+describe('GET api/activities', () => {
+  test('Should return status 200', async () => {
+    const res = await request(app).get('/api/activities').send();
+    expect(res.status).toBe(200);
+    expect(res.body.data).toBeDefined();
+    expect(res.error).toBeFalsy();
+  });
+
+  test('Should return status 404 if the endpoint no exist', async () => {
+    const res = await request(app).get('/api/activitie').send();
```

Comment:
Maybe you can use this test if the endpoint not exists, for all the different methods.